internal/codegen/rest_method.go

```diff
@@ -83,6 +94,53 @@ func (r *Resource) generateGet(m *Method) *Statement {
 	return def
 }
 
+func (r *Resource) generateCreate(m *Method) *Statement {
+	def := Empty()
+	r.addClientFunc(def, m)
+
+	def.BlockFunc(func(def *Group) {
+		var returns []Code
+		// TODO: Support complex keys
+		entity := m.EntityPathKey != nil && m.EntityPathKey.Type.Primitive != nil
+		if entity {
+			returns = append(returns, m.EntityPathKey.Type.Nil())
+		}
+		returns = append(returns, Err())
+
+		m.callResourcePath(def)
+		IfErrReturn(def, returns...).Line()
+		r.callFormatQueryUrl(def)
+		IfErrReturn(def, returns...).Line()
+
+		def.List(Id(ReqVar), Err()).Op(":=").Id(ClientReceiver).Dot("JsonPostRequest").Call(Id(UrlVar), RestLiMethod(protocol.Method_create), Id(CreateParam))
```

Comment:
This line is a little long, can you break it up please? For legibility I sometimes slightly exceed the 120 mark but this is 150 characters. Maybe something like this:
```go
def.List(Id(ReqVar), Err()).
    Op(":=").
    Id(ClientReceiver).Dot("JsonPostRequest").
    Call(Id(UrlVar), RestLiMethod(protocol.Method_create), Id(CreateParam))
```
This is also fine:
```go
def.List(Id(ReqVar), Err()).
    Op(":=").
    Id(ClientReceiver).Dot("JsonPostRequest").Call(Id(UrlVar), RestLiMethod(protocol.Method_create), Id(CreateParam))
```